component/src/main/java/org/cbioportal/genome_nexus/component/annotation/NotationConverter.java

```diff
@@ -163,20 +163,37 @@ public String genomicToHgvs(GenomicLocation genomicLocation) {
             } catch (NumberFormatException e) {
                 return null;
             }
-        } else if (var.equals("-") || var.length() == 0) {
+        } else if (var.equals("-") || var.length() == 0 || var.equals("NA") || var.contains("--")) {
```

Comment:
I'm not sure what a "--" substring in the variant allele means. Could someone comment on the meaning, and the appropriateness of always interpreting such a string as a deletion / indel?

Reply:
it was error in our data on cBioPortal, see also https://github.com/genome-nexus/genome-nexus/issues/276 and https://github.com/cBioPortal/datahub/issues/621. It might be fixed now tho

Reply:
hmm actually i see it in our private databases when running the query, so reopened the issue

Reply:
I have added a note in the datahub issue that once the data has been cleaned up, the special cases for NA and '--' in the notation converter code should be removed. If those are cases which should not exist, I wonder if we can remove those cases from notationconverter now?

Reply:
I guess it doesn't hurt to have these before the issue is fixed, but we could fix it along with the nesting like Ino's comments above